docs/logical_data_model.md

```diff
@@ -188,6 +194,8 @@ Grantee ||--|{ Grant
 Region ||--|{ Grant
 ActivityReport .. ActivityReportCollaborator
 User .. ActivityReportCollaborator
+ActivityReport .. ActivityReportGoal
+Goal .. ActivityReportGoal
```

Comment:
Should there also be a direct Goal <-> ActivityReport connection, similar to Goal <-> Topic?